css/index.css

```diff
@@ -143,4 +239,133 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/* ABOUT PAGE CSS */
+hr {
+    border: 1px dashed;
+}
+
+hr.about {
+    width: 790px;
+}
+
+.about-text {
+    width: 700px;
+    margin: 20px auto;
+}
+
+.container.about-page {
+    width: 700px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.content-box img {
+    width: 100%;
+}
+
+.content-box {
+    margin: 20px 0;
+    width: 48%;
+}
+
+.content-box h3, .content-box p {
+    margin: 10px 0;
+}
+
+button {
+    cursor: pointer;
+}
+
+.learn-more-btn {
```

Comment:
great button and corresponding hover effect, you will learn how to use LESS this week to make this much easier to implement and utilize over and over